src/containers/inAppPurchaseContainer.tsx

```diff
@@ -107,7 +107,7 @@ class InAppPurchaseContainer extends Component {
         };
 
         // make sure item is not consumed if email and username not preset for 999accounts
-        if ((purchase.productId === '999accounts') && (!email || !username)) {
+        if (purchase.productId === '999accounts' && (!email || !username)) {
```

Comment:
Linter issue again? @noumantahir

Reply:
no, this one should be fine.... this should translate to `( boolean1 && ( boolean2 || boolean3 ))` and that is okay